src/app/search/search.component.html

```diff
@@ -82,7 +82,7 @@
           </div>
           <div *ngFor="let subBucket of getKeys(orderedBuckets.get(key).Items); let i = index">
             <div class="panel-container-label">
-              <div *ngIf="i < 5" fxLayout="row" fxLayoutAlign="space-between center">
+              <div *ngIf="i < 5" fxLayout="row" fxLayoutAlign="space-between center" id="facet-row">
```

Comment:
An id [should be unique](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) in the whole document. If I'm reading this right, because this is inside an `ngFor`, you will end up with lots of duplicate ids.

Reply:
Removed this id and instead tagged the #resetButton as there is only one. The button is pretty much the same width as the facet row so using that to determine the histogram width works the same